bemlint/bin/bemlint.js

```
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const { Command } = require('commander');
+const { version } = require('../package.json')
+const { bemlint } = require('../lib/bemlint');
+const path = require('path');
+
+const program = new Command();
+
+program
+  .name('bemlint')
+  .version(version, '-v --version', 'output current version');
+
+program
+  .option('[files...]')
+  .description('check BEM in html')
+  .action((_, destination) => (
+    bemlint(path.join(process.cwd(), ...destination.args)))
```

you need to throw an error if there are errors